packages/cli/lib/LocalDevManager.js

```diff
@@ -510,23 +509,24 @@ class LocalDevManager {
   }
 
   async startServers() {
-    // TODO spin up local dev servers
-    return true;
+    this.devServerPath = await DevServerManager.start({
```

Comment:
Should we be declaring this above line 166, where we reference `this.devServerPath`?

Reply:
This `startServers` function actually [does get called before the code on line 166 executes](https://github.com/HubSpot/hubspot-cli/pull/850/files#diff-0391b654b5bab2f8d2b05906b406db63c1c591cc6b937dfba92d6a7d475cf878R93). So we should be all set there